### PR TITLE
[cmake] simplify copies of misc files into build directory

### PR DESCRIPTION
**Pull request by linev**
Instead of handling each individual file from `tutorials/` or `etc/` or `js/` subdirectory, 
just invoke `cmake -E copy_directory <src> <dst>` command. 
It ignores all files diffs and just copy them. 
Necessary to correctly handle incremental builds

Significantly reduce number of `cmake` invocations - compared with #9804.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -283,28 +283,26 @@ add_custom_command(OUTPUT ${stamp_file}
                    COMMAND ${CMAKE_COMMAND} -E touch ${stamp_file}
                    COMMENT "Copying directories such as etc, icons, fonts, js, ui5, etc. to build area")
 
+#---- List of directories which completely copied to target directory by each build
```
Comment (amadio) [inline CMakeLists.txt:286]:
```suggestion
#---- List of source directories which are copied verbatim into the build directory
```

File: CMakeLists.txt [hunk 2]
```diff
@@ -283,28 +283,26 @@ add_custom_command(OUTPUT ${stamp_file}
...
+set(artifact_dirs "etc;test;icons;fonts;macros;tutorials")
```
Comment (amadio) [inline CMakeLists.txt:287]: You can probably also move the `README` directory from above (line 282) into this list.

**Review by bellenot — COMMENTED** [review]
I agree with @amadio comments, and I prefer this solution over the previous one :-) Thanks to both of you!

File: CMakeLists.txt (hunk 2 above)
Comment (linev) [reply CMakeLists.txt]: Done

**Review by amadio — APPROVED** [review]

**Review by bellenot — APPROVED** [review]

**Pull request merged**